config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # get "/mechanics/:id", to: "mechanics#show"
+
+  resources :mechanics, only: [:show, :update]
+  resources :amusement_parks, only: [:show]
```

Comment:
Nice restful routes. Line 6 would have worked as well, but I like that you used the resources syntax.